pkg/nfd-master/nfd-master.go

```diff
@@ -496,7 +515,7 @@ func addNs(src string, nsToAdd string) string {
 	if strings.Contains(src, "/") {
 		return src
 	}
-	return filepath.Join(nsToAdd, src)
+	return path.Join(nsToAdd, src)
```

Comment:
ran this on windows ?

nit: not directly related to this commit

Reply:
> nit: not directly related to this commit

True. Reverted this change here and addressed in a separate PR (#435)